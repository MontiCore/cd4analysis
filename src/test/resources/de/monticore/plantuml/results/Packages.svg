<?xml version="1.0" encoding="UTF-8" standalone="no"?><svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" contentScriptType="application/ecmascript" contentStyleType="text/css" height="289px" preserveAspectRatio="none" style="width:275px;height:289px;background:#000000;" version="1.1" viewBox="0 0 275 289" width="275px" zoomAndPan="magnify"><defs/><g><rect fill="#FFFFFF" height="27.6094" rx="5" ry="5" style="stroke: #000000; stroke-width: 1.0;" width="102" x="157" y="8"/><text fill="#000000" font-family="sans-serif" font-size="14" lengthAdjust="spacingAndGlyphs" textLength="90" x="163" y="27.5332">.Packages CD</text><text fill="#33FF02" font-family="sans-serif" font-size="14" font-weight="bold" lengthAdjust="spacingAndGlyphs" textLength="140" x="5" y="63.6094">Dot Executable: null</text><text fill="#33FF02" font-family="sans-serif" font-size="14" font-weight="bold" lengthAdjust="spacingAndGlyphs" textLength="173" x="5" y="81.2188">No dot executable found</text><text fill="#33FF02" font-family="sans-serif" font-size="14" font-weight="bold" lengthAdjust="spacingAndGlyphs" textLength="258" x="5" y="98.8281">Cannot find Graphviz. You should try</text><text fill="#33FF02" font-family="sans-serif" font-size="14" font-weight="bold" lengthAdjust="spacingAndGlyphs" textLength="0" x="9" y="116.4375"/><text fill="#33FF02" font-family="sans-serif" font-size="14" font-weight="bold" lengthAdjust="spacingAndGlyphs" textLength="71" x="5" y="134.0469">@startuml</text><text fill="#33FF02" font-family="sans-serif" font-size="14" font-weight="bold" lengthAdjust="spacingAndGlyphs" textLength="50" x="5" y="151.6563">testdot</text><text fill="#33FF02" font-family="sans-serif" font-size="14" font-weight="bold" lengthAdjust="spacingAndGlyphs" textLength="66" x="5" y="169.2656">@enduml</text><text fill="#33FF02" font-family="sans-serif" font-size="14" font-weight="bold" lengthAdjust="spacingAndGlyphs" textLength="0" x="9" y="186.875"/><text fill="#33FF02" font-family="sans-serif" font-size="14" font-weight="bold" lengthAdjust="spacingAndGlyphs" textLength="15" x="9" y="204.4844">or</text><text fill="#33FF02" font-family="sans-serif" font-size="14" font-weight="bold" lengthAdjust="spacingAndGlyphs" textLength="0" x="9" y="222.0938"/><text fill="#33FF02" font-family="sans-serif" font-size="14" font-weight="bold" lengthAdjust="spacingAndGlyphs" textLength="201" x="5" y="239.7031">java -jar plantuml.jar -testdot</text><text fill="#33FF02" font-family="sans-serif" font-size="14" font-weight="bold" lengthAdjust="spacingAndGlyphs" textLength="0" x="9" y="257.3125"/><text fill="#888888" font-family="sans-serif" font-size="10" lengthAdjust="spacingAndGlyphs" textLength="200" x="34" y="276.3027">generated with MontiCore using PlantUML</text><!--MD5=[ff026f14df58d3e4dc1d27887fdad276]
@startuml
  skinparam classAttributeIconSize 0
  skinparam legend {
    BorderColor black
    BackGroundColor white
  }
  legend top right
    .Packages CD
  end legend
  namespace someOtherPackage {
    class A {
      {field} int a
    }
    class B
  }

  class C {
    {field} public someOtherPackage.A c
  }

  center footer generated with MontiCore using PlantUML
@enduml

PlantUML version 1.2020.15(Sun Jun 28 13:39:45 CEST 2020)
(GPL source distribution)
Java Runtime: IBM Semeru Runtime Open Edition
JVM: Eclipse OpenJ9 VM
Default Encoding: windows-1252
Language: de
Country: DE
--></g></svg>
