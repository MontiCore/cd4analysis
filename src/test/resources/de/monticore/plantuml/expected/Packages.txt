@startuml
  skinparam classAttributeIconSize 0
  skinparam legend {
    BorderColor black
    BackGroundColor white
  }
  legend top right
    .Packages CD
  end legend
  namespace someOtherPackage {
    class A {
      {field} int a
    }
    class B
  }

  class C {
    {field} public someOtherPackage.A c
  }

  center footer generated with MontiCore using PlantUML
@enduml
